internal/deltastream/aws/deployment-config.go

```diff
@@ -146,6 +146,12 @@ const deploymentConfigTmpl = `
 	"domain": "{{ .DSSecret.Auth0Cli.Domain }}",
 	"clientId": "{{ .DSSecret.Auth0Cli.ClientId }}"
   },
+  "auth0TrialMgmt": {
+	"domain": "{{ .DSSecret.auth0TrialMgmt.Domain }}",
```

Comment:
Typo in DSSecret.auth0TrialMgmt
```suggestion
	"domain": "{{ .DSSecret.Auth0TrialMgmt.Domain }}",
```